client/src/App.js

```diff
@@ -4,18 +4,20 @@ import Product from './pages/product';
 import Cart from './pages/cart';
 import styled from 'styled-components';
 import { tastes } from './database.js';
-import { useState } from 'react';
-
+import { useEffect, useState } from 'react';
 
 function App() {
-  
-  const [cartItems, setCartItems] = useState([]);
+  const [cartItems, setCartItems] = useState(
+    () => JSON.parse(localStorage.getItem('cartItems')) || []
+  );
 
   const onAdd = tasteSample => {
     const existingItem = cartItems.find(item => item.id === tasteSample.id);
     if (existingItem) {
       setCartItems(
-        cartItems.map(item => (item.id === existingItem.id ? { ...existingItem, qty: existingItem.qty + 1 } : item))
+        cartItems.map(item =>
+          item.id === existingItem.id ? { ...existingItem, qty: existingItem.qty + 1 } : item
```

Comment:
You could use a filter and a map. This would be more understandable and easy to read 🐟

Reply:
Hey Julia! I'm afraid I don't quite understand what you mean. Also, because your suggestion looks exactly the same? This part of code had already been reviewed and there were no complaints so far, so may I just leave it like that please? Thanks!

Reply:
sure, you could leave it like this, it is a suggestion :)